L/libcint/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libcint"
+version = v"3.0.20"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/sunqm/libcint.git", "34108d98a0d7f9836c2d218a6ed0e0ba7efff84f")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libcint/
+
+mkdir build && cd build
+
+cmake .. -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBLAS_LIBRARIES="-lopenblas"
+
+make
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:x86_64, libc=:musl),
+    Windows(:x86_64),
+    MacOS(:x86_64),
+    Linux(:x86_64, libc=:glibc)
+]
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libcint", :libcint)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="OpenBLAS32_jll", uuid="656ef2d0-ae68-5445-9ca0-591084a874a2"))
```

Comment:
Does this really require the 32-bit version of OpenBLAS?

Reply:
I believe OpenBLAS32 refers to the 32-bit integer version (LP64). In that case, I think this is the version libcint should link to by default.